omegatrade-app/backend/app/controller/dashboard.controller.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const Company = require('../models/company.model');
+const logService = require('../helpers/logservice');
+
+/**
+ * Function to get stockdata of a company
+ * @param {\} req 
```

Comment:
nit: document what is expected in `req`.

Reply:
The above issue has been resolved.